src/actions.ts

```diff
@@ -290,69 +302,61 @@ export const blur = () => ({
 
 // Key Bindings
 
-type KeyDownHandlers<Cell: Types.CellBase> = {
-  [eventType: string]: KeyDownHandler<Cell>
+type KeyDownHandlers<Cell> = {
```

Comment:
```suggestion
type KeyDownHandlers<Cell extends Types.CellBase> = {
```
IIRC this is equivalent in TS